### ISSUE TITLE
[Bug] AccessLogFilter for path matrics make Server OOM

### ISSUE THREAD

**Issue by JackyYangPassion** [apache/incubator-hugegraph#2566]
### Bug Type (问题类型)

logic (逻辑设计问题)

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/apache/hugegraph/issues) 与 [FAQ](https://hugegraph.apache.org/docs/guides/faq/) 中没有相同 / 重复问题 (I have confirmed and searched that there are no similar problems in the historical issue and documents)

### Environment (环境信息)

- Server Version: 1.3.0 (Apache Release Version)
- Backend: RocksDB  

### Expected & Actual behavior (期望与实际表现)

API
```
PUT http://127.0.0.1:8080/graphs/hugegraph/graph/vertices/"1:mako"?action=append

{
    "label": "person",
    "properties": {
        "age": 30,
        "city": "Beijing"
    }
}

```

Get metrics
```
curl http:/127.0.0.1:8080/metrics |grep vertices

result
_graphs_hugegraph_graph_vertices_%221:mako%22_PUT_RESPONSE_TIME_HISTOGRAM
```

when there is many Vetices update

This Map will be oom
ServerReporter.java
```
    private SortedMap<String, Histogram> histograms;
```

**Comment by JackyYangPassion** [comment]
@dosu  can you help me fix this bug

**Comment by JackyYangPassion** [comment]
The root casue is this class logic

AccessLogFilter.java
```
@Override
    public void filter(ContainerRequestContext requestContext,
                       ContainerResponseContext responseContext) throws IOException {
        // Grab corresponding request / response info from context;
        URI uri = requestContext.getUriInfo().getRequestUri();
        String path = uri.getRawPath();//针对不同的path中包含变量的方法，会造成Metrics 采集 OOM
        String method = requestContext.getMethod();
        String metricsName = join(path, method);
        //TODO: 变量path 造成OOM。暂时先屏蔽，啥都不做
        MetricsUtil.registerCounter(join(metricsName, METRICS_PATH_TOTAL_COUNTER)).inc();
        if (statusOk(responseContext.getStatus())) {
            MetricsUtil.registerCounter(join(metricsName, METRICS_PATH_SUCCESS_COUNTER)).inc();
        } else {
            MetricsUtil.registerCounter(join(metricsName, METRICS_PATH_FAILED_COUNTER)).inc();
        }

        Object requestTime = requestContext.getProperty(REQUEST_TIME);
        if (requestTime != null) {
            long now = System.currentTimeMillis();
            long start = (Long) requestTime;
            long executeTime = now - start;

            MetricsUtil.registerHistogram(join(metricsName, METRICS_PATH_RESPONSE_TIME_HISTOGRAM))
                       .update(executeTime);

            HugeConfig config = configProvider.get();
            long timeThreshold = config.get(ServerOptions.SLOW_QUERY_LOG_TIME_THRESHOLD);
            // Record slow query if meet needs, watch out the perf
            if (timeThreshold > 0 && executeTime > timeThreshold &&
                needRecordLog(requestContext)) {
                // TODO: set RequestBody null, handle it later & should record "client IP"
                LOG.info("[Slow Query] execTime={}ms, body={}, method={}, path={}, query={}",
                         executeTime, null, method, path, uri.getQuery());
            }
        }

        // Unset the context in "HugeAuthenticator", need distinguish Graph/Auth server lifecycle
        GraphManager manager = managerProvider.get();
        // TODO: transfer Authorizer if we need after.
        if (manager.requireAuthentication()) {
            manager.unauthorize(requestContext.getSecurityContext());
        }
    }

```
@dosu could you fix this bug

**Comment by JackyYangPassion** [comment]
@dosu 上边思路是正确的，但是

```
private String normalizePath(String path) {
        // Replace variable parts of the path with placeholders
        return path.replaceAll("/\\d+", "/{id}")
                   .replaceAll("/[a-f0-9-]{36}", "/{uuid}");
    }
```
 方法不能覆盖path = /graphs/hugegraph/graph/vertices/"2:mako"  这种场景，请修正下代码

**Comment by JackyYangPassion** [comment]
fix #2569
